### PR TITLE
SciTools Status Page

### PR DESCRIPTION
**Pull request by LukeC92**
The main change from this request is the creation of a page that shows the build status and test coverage of any SciTools or SciTools-incubator packages we are interested in. It is not linked to from any other page but it does have the links to the rest of the site; this was done by adding build_status.html to build.py. I also added some things that may be helpful for this, if not now then maybe in the future; e.g. project.mobirise, datatables and some css formating.

I also added an example web page that I could use to copy code from, but I removed this on the last commit.

### CODE REVIEW

**Comment by pelson** [comment]
This is the link for the docs: https://rawgit.com/LukeC92/scitools.org.uk/status_page/build_status.html
Unfortunately, it doesn't look right. @LukeC92, could you take a look please?

File: build_status.html [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+                <th class="head-item mbr-fonts-style display-7">
+                  Build Status
+                </th>
+                <th class="head-item mbr-fonts-style display-7">
+                  Test Coverage
+                </th>
+              </tr>
+            </thead>
+            <tbody>
+
```
Comment (pelson) [inline build_status.html]: Perhaps put a html comment in here to highlight that this is populated via JS at the end of the file

File: build_status.html [hunk 2]
```diff
@@ -0,0 +1,353 @@
...
+                      ];
+
+              table = $('#status-table');
+              body = $('tbody')
+
+              function build_row(repo) {
+                     return $('<tr>'
+                            + ' <td class="body-item mbr-fonts-style display-7">' + repo.name + '</td>'
+                            + ' <td class="body-item mbr-fonts-style display-7">'
+                            + '   <a href="https://coveralls.io/github/' + repo.org + '/' + repo.name + '?branch=master">'
```
Comment (pelson) [inline build_status.html]: This is the wrong link.

File: build_status.html [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+                            + '    <img src="https://travis-ci.org/' + repo.org + '/' + repo.name + '.svg?branch=master" alt="iris-build-status" title="iris-build-status">' 
+                            + '   </a>'
+                            + ' </td>'
+                            + ' <td class="body-item mbr-fonts-style display-7">'
+                            + '   <a href="https://coveralls.io/github/' + repo.org + '/' + repo.name + '?branch=master">'
+                            + '     <img src="https://coveralls.io/repos/github/' + repo.org + '/' + repo.name + '/badge.svg?branch=master" alt="Iris Coverage Status" />'
+                            + '   </a>'
+                            + ' </td>'
```
Comment (pelson) [inline build_status.html]: Could you add the other badges like the ones at https://github.com/scitools/cf_units.

**Comment by LukeC92** [comment]
I think I have addressed all of the issues previously brought up. Though it's worth noting that having "?branch=master" on the end of many of the links doesn't seem to make a difference. It seems to make a difference to most, if not all, of the images. But the hrefs seem to go to the same place regardless of the branch specified. Hence I have remove these parts of the URLs. 

It may be that this makes a difference for some of the links and others just need a different format.

**Comment by pelson** [comment]
This is fabulous @LukeC92! It is so useful!!! I'm super happy to have this 😄 🎉 🥇

**Comment by pelson** [comment]
@SciTools/developers - this really neat page just landed: take a look at https://scitools.org.uk/build_status.html

We haven't integrated it into the scitools.org.uk pages yet, but those with the secret-sauce (the url... that isn't that secret) can see it. I for one will be checking this around once a week to ensure everything remains tickety-boo on our projects. It also highlights that I need to get back to fixing those pesky tests... 😜

**Comment by lbdreyer** [comment]
@LukeC92 I suspect it is due to the shields badges not correctly understanding how to handle release candidates, so that it thinks v2.0.0rc1 is > v2.0.0 (See this comment: https://github.com/badges/shields/issues/490#issuecomment-205550621)

We could potentially raise it with the shields developers

**Pull request merged**